.releaserc

```diff
@@ -1,15 +1,18 @@
 {
   "branch": "master",
-  "verifyConditions": [],
+  "verifyConditions": ["@semantic-release/changelog", "@semantic-release/github"],
+  "debug": true,
```

Comment:
probably don't need the "debug" field here, but also ok to leave it you want... I think it just results in more output in the travis build log